.github/workflows/build_rpms.yml

```diff
@@ -0,0 +1,50 @@
+name: RPM Build
+
+on: [pull_request]
```

Comment:
I think you want `push` as well, to make sure updated PRs run this as well:

`on: [push, pull_request]`